docs/source/FAQ.md

```diff
@@ -477,3 +477,62 @@ Training options to perform vocabulary update are:
 * `-update_vocab`: set this option
 * `-reset_optim`: set the value to "states"
 * `-train_from`: checkpoint path
+
+
+## How can I use source word features?
```

Comment:
I think we should give some pointers about how to use it with Transformer here. It's not very straightforward and may trigger a lot of questions from users.
Some things to mention :
- `feat_vec_size`, `src_word_vec_size` and `tgt_word_vec_size` should be correctly set;
- not possible to do shared embeddings (at least with concat method)
- something else?